README.md

```diff
@@ -89,6 +93,16 @@ const logstashConf: LogstashConfiguration[] = [
     },
 ];
 
+/*
+Use this if you want that the library will decide what protocol to use(TCP or UDP).
```

Comment:
"which" instead of "what".
also, like the comment above, mention that this is not only for tcp/udp configurations